### ISSUE TITLE
Bringing incorrect data after google play ui update

### ISSUE THREAD

**Issue by lucasknopp** [n0madic/google-play-scraper#8]
I would like to know if it will be remapped to a new interface or if there is a solution to this problem

`{
  "AdSupported": false,
  "AndroidVersion": "[[null,[[1800],null,[72000]]]]",
  "AndroidVersionMin": 0,
  "ContentRating": "",
  "ContentRatingDescription": "",
  "Description": "",
  "DescriptionHTML": "",
  "Developer": "",
  "DeveloperAddress": "",
  "DeveloperEmail": "",
  "DeveloperID": "",
  "DeveloperURL": "https://play.google.com",
  "DeveloperWebsite": "",
  "FamilyGenre": "",
  "FamilyGenreID": "",
  "Free": true,
  "Genre": "",
  "GenreID": "",
  "HeaderImage": "",
  "IAPOffers": false,
  "IAPRange": "",
  "Icon": "",
  "ID": "com.whatsapp",`

**Comment by n0madic** [comment]
Yes, work in progress, but not all properties have been restored yet

**Comment by n0madic** [comment]
[Fixed](https://github.com/n0madic/google-play-scraper/commit/8b86b7d3198df754f131d52ca1aafdab480a7431) app mapping

Breaking change: removed Size property

**Comment by lucasknopp** [comment]
Thank you excellent work!

**Comment by ismael-albert** [comment]
top d+
